g/GDScript.gd

```diff
@@ -1,4 +1 @@
-extends Label
-
-func _ready():
-  self.text = "Hello World"
+print("Hello World")
```

Comment:
Shouldn't this be executed in some sort of init function?

https://docs.godotengine.org/en/latest/getting_started/step_by_step/scripting_first_script.html

Reply:
Yes, I suggest this:

```suggestion
extends Node2D

func _ready():
    print("Hello World")

```

Even though `Node` is the core class, `Node2D` should be used for most use cases, and people use Godot for 2D games mostly.